python/pyspark/ml/feature.py

```diff
@@ -3765,6 +3765,15 @@ def labels(self):
         """
```

Comment:
I think it's fine to add the deprecation note here together but cc @srowen.

Reply:
It is okay to have a deprecation note. But the version might be changed to 3.0.2 too.

Reply:
I will leave this one to @srowen as well but I believe that should be fine as this one itself is an exception to add an API to maintenance release. It doesn't break anything and just add one line in the documentation that should have been done for Spark 3.0.0. I think this is more like a matter of judgement.

Reply:
Adding the deprecation note to labels? yeah that could be fine, but not essential here. Either way.